src/genpy/rostime.py

```diff
@@ -398,6 +398,8 @@ def __floordiv__(self, val):
             return Duration(self.secs // val, self.nsecs // val)
         elif t == float:
             return Duration.from_sec(self.to_sec() // val)
+        elif isinstance(val, Duration):
+            return self.to_sec() // val.to_sec()
```

Comment:
Note that unlike everything else mentioned in #49, `Duration // Duration` _is_ meaningful